nibabel/openers.py

```diff
@@ -9,19 +9,66 @@
 """ Context manager openers for various fileobject types
 """
 
-from os.path import splitext
-import gzip
 import bz2
```

Comment:
That was Python 2.7, but good point--I didn't use Python3. Python 3 is hanging, using a ton of resources.